tools/docker-utils.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const tempy = require('tempy');
+const execa = require('execa');
+const { getUnleashServerVersion } = require('./unleash-server-utils');
+
+const tempDirPath = tempy.directory();
+
+/** @type {string} */
+const dockerfileTemplate = fs.readFileSync(
+  path.resolve(__dirname, '..', 'Dockerfile'),
+  { encoding: 'utf8' },
+);
```

Comment:
I load this file sync outside the other functions to avoid loading it more than once. Either way is probably fine. Do you think this is okay?

Reply:
Should be fine. It's not like it's several MBs